From 45dc1808778513e467ad73db90c427c4711994a4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Corentin=20No=C3=ABl?= <corentin.noel@collabora.com>
Date: Fri, 14 Aug 2020 16:47:34 +0200
Subject: [PATCH] gegl: Explicit the vapi dependencies

This makes sure that projects using gegl are also including glib-2.0, gobject-2.0 and babl-0.1.
---
 gegl/gegl.deps.in | 0
 gegl/meson.build  | 5 +++++
 2 files changed, 5 insertions(+)
 delete mode 100644 gegl/gegl.deps.in

diff --git a/gegl/gegl.deps.in b/gegl/gegl.deps.in
deleted file mode 100644
index e69de29bb..000000000
diff --git a/gegl/meson.build b/gegl/meson.build
index c92110349..4eee76810 100644
--- a/gegl/meson.build
+++ b/gegl/meson.build
@@ -137,6 +137,11 @@ if get_option('introspection')
       metadata_dirs: '.',
       sources: gegl_gir[0],
       install: true,
+      packages: [
+        'glib-2.0',
+        'gobject-2.0',
+        'babl-0.1'
+      ],
     )
   endif
 endif
